developers/weaviate/concepts/vector-index.md

```diff
@@ -28,9 +30,11 @@ Weaviate's vector-first storage system takes care of all storage operations with
 
 ## Why index data as vectors?
 
-Vectors are a great way to represent data objects' meaning, such as meaning of text or contents of images or video. They also enable determination of [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) between data objects.
+Vectors are a great way to represent data objects' meaning, such as meaning of text or contents of images or video. They also enable determination of [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) between data objects, which enables vector databases to retrieve data objects based on their similarity to a query vector - also called similarity search.
+
+To make this concept more tangible, think of vectors as coordinates in a *n*-dimensional vector space. For example, we can represent *words* in a 2-dimensional space. If you use an algorithm that learned the relations of words or co-occurrence statistics between words from a corpus (like [GloVe](https://github.com/stanfordnlp/GloVe)), then single words can be given the coordinates (vectors or [vector embeddings](https://weaviate.io/blog/vector-embeddings-explained#what-exactly-are-vector-embeddings)) according to their similarity to other words. These algorithms are powered by Machine Learning and Natural Language Processing concepts. 
```

Comment:
Consider:

Vectors are like coordinates in a multi-dimensional space. A very simple vector might represent objects, words in this case, in a 2-dimensional space.

Reply:
included